cmd/node/node.go

```diff
@@ -171,7 +171,7 @@ type TickProvider interface {
 	GetCurrentLayer() types.LayerID
 	StartNotifying()
 	GetGenesisTime() time.Time
-	LayerToTime(types.LayerID) time.Time
+	timesync.LayerConverter
```

Comment:
I agree that this interface is abused. However, `LayerConverter` is indirectly part of this interface already (via its `LayerToTime` method). I only substituted it with the whole interface to also include the `DurationToLayers` method.

This interface needs ground refactoring which is out of the scope of this PR.

Reply:
Then please create a ticket for it. `TickProvider` and `LayerConverter` should be their own independent interfaces with their own implementations (which is already the case). Just the users of `TickProvider` should use `LayerConverter` directly instead of via `TickConverter`.

I just think implementing this change takes hardly longer than writing a ticket for it.

Reply:
I created a new tech-debt ticket to address it: https://github.com/spacemeshos/go-spacemesh/issues/3621